releasenotes/notes/1.3/add-gates-to-collect-clifford-af88dd8f7a2a4bf9.yaml

```
@@ -1,7 +1,7 @@
 ---
 features_transpiler:
   - |
-    Add an argument ``matrix_based`` to the :class:`.CollectCliffords()` transpiler pass.
+    Add argument ``matrix_based`` to the :class:`.CollectCliffords()` transpiler pass.
```

I'm not sure on this one, but I thought this shouldn't include the brackets? Below there's some more occurences and in particular 
```
:class:`.RZGate(np.pi/2)`
```
which looks sus -- but it might be more likely that I just don't know about this option 🙂 

```suggestion
    Add argument ``matrix_based`` to the :class:`.CollectCliffords` transpiler pass.
```